src/share/routes.js

```diff
@@ -78,6 +78,26 @@ function register(router) {
 
         res.send(note.getContent());
     });
+    
+    router.get('/share/api/notes/:noteId/view', (req, res, next) => {
+        const {noteId} = req.params;
+        const note = shaca.getNote(noteId);
+
+        if (!note) {
+            return res.status(404).send(`Not found`);
+        }
+
+        const utils = require("../services/utils");
+
+        const filename = utils.formatDownloadTitle(note.title, note.type, note.mime);
+
+        // res.setHeader('Content-Disposition', utils.getContentDisposition(filename));
```

Comment:
I left this commented out to better show the difference between the two API endpoints, and to clearly show that this does not send that header.